ranger-discovery-bundle/src/test/java/io/appform/ranger/discovery/bundle/ServiceDiscoveryBundleHierarchicalSelectorTest.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright (c) 2019 Santanu Sinha <santanu.sinha@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package io.appform.ranger.discovery.bundle;
```

Comment:
Tests seems to be placed in a flat structure. Please move them into proper places

Reply:
The test is not for HST, it is for SDBundle with HST, similar to SDBundleRotationTest, SDBundleDwMonitorTest and so on. The HST test is properly located in its own package.